azure_monitor/src/azure_monitor/utils.py

```diff
@@ -88,7 +99,6 @@ class Options(BaseObject):
     def __init__(
         self,
         connection_string=None,
-        endpoint="https://dc.services.visualstudio.com/v2/track",
```

Comment:
Why change this logic?

Reply:
endpoint would be overridden with connection string logic so why allowing the user to specify it, do we want to support this?, if we do which config have more priority specific endpoint in Options object or endpoint in ConnectionString

Reply:
That's true. We can remove endpoint.